meta-resin-common/recipes-support/resin-vars/resin-vars/os-networkmanager

```diff
@@ -91,6 +91,30 @@ if [ "$config_json_value" != "null" ]; then
 	nm_config="$(printf "%s\nwifi.scan-rand-mac-address=%s" "$nm_config" "$value")"
 fi
 
+# [connectivity]
+section="connectivity"
+s=0
```

Comment:
I might be missing something but it looks like this variable is unused?

Reply:
removing